pkg/config/validate.go

```diff
@@ -336,17 +313,8 @@ func mergeBlueprintRequirements(base map[string][]string, new map[string][]strin
 	return dest
 }
 
-func (dc *DeploymentConfig) testApisEnabled(validator validatorConfig) error {
-	requiredInputs := []string{}
-	funcName := testApisEnabledName.String()
-	funcErrorMsg := fmt.Sprintf(funcErrorMsgTemplate, funcName)
-
-	if validator.Validator != funcName {
-		return fmt.Errorf("passed wrong validator to %s implementation", funcName)
-	}
-
-	err := testInputList(validator.Validator, validator.Inputs, requiredInputs)
-	if err != nil {
+func (dc *DeploymentConfig) testApisEnabled(c validatorConfig) error {
```

Comment:
nit: I am curious on the decision to go from `validator` to `c`. I know short arg names is idiomatic for go. `validator` could cause confusion because it is so close to `validators` which is something very different. But maybe `config` is more readable than `c`. No strong opinions here.

Reply:
Having written much of this code, I'll say that I `+1` disambiguating the several uses of `validate`, `validator`, and `validatorConfigs` here.

Reply:
My intent is `c` is for `Config`. I would keep it `c` to follow golang code style.
@tpdownes  do you have any concerns? I guess `c` does disambiguate it somewhat.

Reply:
LGTM